manim/animation/creation.py

```diff
@@ -200,6 +203,105 @@ def __init__(
         super().__init__(mobject, rate_func=rate_func, remover=remover, **kwargs)
 
 
+class Add(Animation):
+    """Adds a Mobject to the scene with no animated effect. This is functionally equivalent to :meth:`~.Scene.add`
+
```

Comment:
But uuh what's the differnce between this and usig `Animation` ?

Reply:
Does just `Animation(mob)` do the same thing as `Add`? If so, that was never clear to me and I wouldn't expect users to know this is possible.

Reply:
you're probably right. Furthermore, it's a step forward making Animation abstract

But yes, it does exactly what Add does because Add inherits from Animation without redefining anything.

Reply:
It does act the same. Though I am confused why as I do not see anywhere in the Animation class that adds the mobject to the scene. I changed it now so that Add inherits from Animation and does nothing. I think we should make Animation more abstract as you said, and having Animation effectively work as Add is odd behavior that we shouldn't expect the user to know.

Reply:
Also, I'm not sure what the problem with calling it Add is? It's quite literally what it is -- just like `self.add()`

Reply:
Yes, there is no problem - I just don't like the name `Add` but that's a matter of taste ;) 

> Though I am confused why as I do not see anywhere in the Animation class that adds the mobject to the scene.

https://github.com/huguesdevimeux/manim/blob/23f5b643a112cce10b639b5759307b3cecfc3621/manim/scene/scene.py#L364-L373